src/test/java/hudson/tasks/junit/JUnitResultArchiverTest.java

```diff
@@ -482,16 +484,24 @@ public void testXxe() throws Exception {
         String oobInUserContentLink = j.getURL() + "userContent/oob.xml";
         String triggerLink = j.getURL() + "triggerMe";
 
-        String xxeFile = this.getClass().getResource("testXxe-xxe.xml").getFile();
-        String xxeFileContent = FileUtils.readFileToString(new File(xxeFile), StandardCharsets.UTF_8);
+        URL xxeResourceUrl = this.getClass().getResource("testXxe-xxe.xml");
```

Comment:
Are these changes related to your change? You can submit them separately with reasoning if not